docs/sources/lids/0001-Introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction of LIDs
+
+**Author:** Danny Kopping (danny.kopping@grafana.com)
+
+**Date:** 01/2023
+
+**Sponsor(s):** @dannykoppping
+
+**Type:** Process
+
+**Status:** In Discussion
+
+**Related issues/PRs:** N/A
+
+**Thread from [`#loki-dev` channel on Slack](https://slack.grafana.com):** N/A
+
+---
+
+## Background
+
+As the Grafana Loki project grows, we have seen more and more contributions from external (outside Grafana Labs) contributors.
+
+## Problem Statement
+
+Many of these external contributions are large and complex, and have taken these contributors significant time to implement. Large contributions that are made without prior discussion with maintainers are at risk of being rejected if they are misguided, implemented inefficiently, or simply undesired; this is obviously suboptimal both for the contributors and the maintainers.
+
+Aside from external contributions, changes being proposed by Grafana Loki maintainers may also require community engagement before being worked on.
+
+## Goals
+
+It would be preferable to engage with contributors _before_ they make large contributions to ensure that both their and the project's interests are aligned. The community at large must also have a voice when feature or process changes are being proposed, to protect their own interests.
+
+We should implement a **lightweight** process that guides the implementation of major changes to the project.
+
+## Proposals
+
+### Proposal 0: Do nothing
+
+We will continue to attract large, often complex, external contributions that have not be discussed with maintainers prior to the work being put in; this may lead to suboptimal outcomes for the relationship between the project and its community.
+
+### Proposal 1: Loki Improvement Documents
```

Comment:
What about using Github issues instead? I do like the idea of LID but wonder where they fit in. Also, does this mean we drop design documents?

Reply:
Design documents (referring to our internal Grafana Labs process) would still remain in place. We would probably use that process internally to gain consensus, and then put it into this format for public visibility & community comment; I've kept the formats pretty similar to make that easy.

GitHub Issues are not ideal because of the thread model of conversation. Inline threads are far more useful than a flat list of threads when it comes to detail-heavy topics IMHO.